src/project-management/release-train-model.md

```diff
@@ -0,0 +1,54 @@
+    2. If a feature misses a release train we have a good idea of when it will show up.
+    3. Users know when to expect their features
+3. Transparency. There will be a published cut-off date (AKA code freeze) for the release and people will know about it in advance. Hopefully this will remove the contention around which features make it.
+4. Quality. we've seen issues pop up in release candidates due to last-minute features that didn't have proper time to bake in. More time between code freeze and release will let us test more, document more and resolve more issues.
+5. Project visibility and activity. Having frequent releases improves our visibility and gives the community more opportunities to talk about TiDB.
+
+Since nothing is ever perfect, there will be some downsides:
+
+1. Most notably, features that miss the code-freeze date for a release will have to wait few months for the next release train. Features will reach users faster overall as per benefit #1, but individual features that just miss the cut will lose out.
+2. For users, figuring out which release to use and having frequent new releases to upgrade to may be a bit confusing.
+3. Frequent releases mean we need to do bugfix releases for older branches.
+
+We decided to experiment with release train model and see if the benefits for us as a community exceed the drawbacks. 
+
+## How will TiDB development process look like?
+
+At this stage we are planning to make a release every two months.
+
+Thus, a typical development cycle takes two months, which we call a sprint. For example, the development cycle of v5.2 is from the end of June to the end of August and called Sprint 4.
+
+Two weeks before the release date, the release manager will cut branches, publish a list of features that will be included in the release, and also announce code-freeze, after which only fixes for blocking bugs will be merged. This announcement will be posted on the [TiDB Internals forum](https://internals.tidb.io/).
+
+For release train model, we will strictly ensure that a release happens on a given date. For example, in the release of v5.2, we will decide to have a release by the end of August and we will stick to it. We will drop features that we think will not make it into the release at the time of code freeze and also avoid taking on new features into the release branch. Master development can continue as usual and those features will be in the following release. 
+
+Ideally, we would have started stabilization around this time. After code freeze, pull requests to the release branch will be merged only if any blocker bugs are identified. In a rare scenario, we could end up with a feature that passed the code freeze bar but still fails to complete on time. Such features will also be dropped from the release at the end to meet the release deadline.
+
+## What happens if features don't complete?
```

Comment:
```suggestion
## What happens to features that are not completed yet?
```

Reply:
It is a bit confused by "yet". I tend to keep the original expression.

Reply:
it sounds inappropriate to say " features don't complete", because "feature" is a noun that cannot do things like humans.  How about changing this title to  " What happens if features are not completed"?